openmetadata-ui/src/main/resources/ui/src/interface/teamsAndUsers.interface.ts

```diff
@@ -114,10 +115,17 @@ export interface TeamDetailsProp {
   handleLeaveTeamClick: (id: string, data: Operation[]) => Promise<void>;
   childTeams: Team[];
   showDeletedTeam: boolean;
+  onAssetsPaginate: (page: string | number) => void;
   onShowDeletedTeamChange: (checked: boolean) => void;
   onTeamExpand: (
     loading?: boolean,
     parentTeam?: string,
     updateChildNode?: boolean
   ) => void;
 }
+
+export interface EntitiesType {
+  data: FormattedTableData[];
+  total: number;
+  currPage: number;
+}
```

Comment:
Not sure of this type. We should avoid creating custom types if not necessary as it will become tedious to maintain. See if we could use some of existingg type

Reply:
Done